commandline.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2017 Ontario Systems
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package isclib
+
+import (
+	"os/exec"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// Commands represents the ISC command lines that are available
+type Commands uint
+
+func (c *Commands) Set(command Commands)     { *c |= command }
+func (c *Commands) Clear(command Commands)   { *c &^= command }
+func (c *Commands) Toggle(command Commands)  { *c ^= command }
```

Comment:
Is there a case where `Toggle` is used? I can't think of a case where you would want to just swap the existence of a command to its opposite. I only ask because it's being made part of the public API which means you have to take it into account for API breaks etc. So, if it's not necessary, I would remove it before this is released.

Reply:
Removed